src/php/Compiler/Analyzer/TypeAnalyzer/SpecialForm/NsSymbol.php

```diff
@@ -172,4 +175,14 @@ private function analyzeRequire(string $ns, PersistentListInterface $import): Sy
 
         return $requireSymbol;
     }
+
+    private function analyzeRequireFile(PersistentListInterface $import): string
+    {
+        $file = $import->get(1);
```

Comment:
What's the meaning of this `1`? Maybe would be better extracting that into a constant or writing a comment above giving some explanation about that number? 🤔

Reply:
`$import` is something like `(require-file "vendor/autoload.php")`. `$import->get(1)` is then `"vendor/autoload.php". Don't know if we can change this.